### PR TITLE
refactor: change pynput to oa_pynput

### PR DESCRIPTION
**Pull request by Mustaballer**
- Changed all occurences of pynput in python files to oa_pynput.
- created README.md for pypi

### CODE REVIEW

**Comment by Mustaballer** [comment]
@abrichr Ready for Review! Once this is merged, I will upload a new release to pypi, which ideally should solve the issues when running `poetry install oa-pynput`

File: setup.py [hunk 1]
```diff
@@ -6,16 +6,16 @@
...
-MAIN_PACKAGE_NAME = 'pynput'
+MAIN_PACKAGE_NAME = 'oa_pynput'
 
 #: The package URL
-PACKAGE_URL = 'https://github.com/moses-palmer/pynput'
+PACKAGE_URL = 'https://github.com/OpenAdaptAI/pynput'
 
 #: The author email
-AUTHOR_EMAIL = 'moses.palmer@gmail.com'
+AUTHOR_EMAIL = 'richard.abrich@mldsai.com'
```
Comment (abrichr) [inline setup.py]: Let's leave this as it was

**Comment by abrichr** [comment]
Looks good! Just one small suggestion 😄

File: setup.py (hunk 1 above)
Comment (Mustaballer) [reply setup.py]: addressed in https://github.com/OpenAdaptAI/pynput/pull/3/commits/7e36c55e2ea0ccc47d0a8ca37487f002edd0027f

**Pull request merged**